tablecodec/tablecodec.go

```diff
@@ -469,6 +469,16 @@ func appendTableIndexPrefix(buf []byte, tableID int64) []byte {
 	return buf
 }
 
+// ReplaceRecordKeyTableID replace the tableID in the recordKey buf.
+func ReplaceRecordKeyTableID(buf []byte, tableID int64) []byte {
+	if len(buf) < len(tablePrefix)+8 {
+		return buf
+	}
+
+	codec.PutInt(buf[len(tablePrefix):], tableID)
```

Comment:
How about using `EncodeInt` and `copy`, then we don't need to add a new API.

Reply:
use PutInt can avoid memcpy. and bytes array append